pkgs/top-level/all-packages.nix

```diff
@@ -1550,6 +1550,8 @@ in
 
   libbladeRF = callPackage ../development/libraries/libbladeRF { };
 
+  libgpuarray-cuda = python34Packages.libgpuarray-cuda;
+
```

Comment:
is libgpuarray-cuda supposed to be used as a Python library, as an application, or a library in general (not Python specific). We only use these kind of references in the case of applications.